src/Sulu/Bundle/PageBundle/Controller/AbstractExtensionController.php

```diff
@@ -56,13 +59,13 @@ public function cgetAction()
         throw new NotFoundHttpException();
     }
 
-    public function getAction(Request $request, $uuid)
```

Comment:
???

Reply:
The name of this variable is also the name of the route parameter. If I would leave it to `uuid`, then I would have to handle both version (`uuid` and `id`) in the javascript code.